utilities/utility.py

```diff
@@ -12,6 +12,36 @@ def validate_empty_fields(**kwargs):
             raise AttributeError(field + " is required field")
 
 
+def validate_country_field(**kwargs):
+    """
+    Function to validate country fields when
+    saving an object
+    :params kwargs
+    """
+    my_args = []
+    countries = ['Kenya', 'Uganda', 'Nigeria']
+    for value in kwargs:
+        country_args = kwargs.get(value)
+        my_args.append(country_args)
+    if my_args[2] not in countries:
+        raise AttributeError("Not a valid country")
+
+
+def validate_timezone_field(**kwargs):
+    """
+    Function to validate country fields when
+    saving an object
+    :params kwargs
+    """
+    my_args = []
+    timezones = ['EAST_AFRICA_TIME', 'WEST_AFRICA_TIME']
+    for value in kwargs:
+        time_zone_args = kwargs.get(value)
+        my_args.append(time_zone_args)
+    if my_args[3] not in timezones:
```

Comment:
You have an error in this validator, remember you only have 2 items in the list, and you looking for one with inex of 3 which is ideally the 4th item, this will always fail!

Reply:
let me check it out  now.

Reply:
@shemogumbe  it okay

Reply:
its not an error
<img width="456" alt="screen shot 2018-07-10 at 18 46 54" src="https://user-images.githubusercontent.com/29824008/42521468-a71f2174-8471-11e8-9235-832ef5bf6cd3.png">
Am picking form this list of arguments not the enumerators

Reply:
And btw, how many arguments should we pass this validator function, though just one would be enough, would it be necessary to pass multiple ?
I think we need to:
1. Pass just one argument.
2. it should even not be a keyword argument